ocs_ci/deployment/acm.py

```diff
@@ -249,3 +249,35 @@ def get_default_gateway_node(self):
         """
         # Always return the first worker node
         return get_typed_worker_nodes()[0]
+
+
+def download_subctl_binary(source="upstream"):
+    if source == "upstream":
```

Comment:
Maybe we can add else block with raising exception if the other source will be provided as we currently support only the `upsteram`.